data/prod/news/c0000000025o.json

```diff
@@ -443,7 +443,7 @@
                   "height": 576,
                   "locator": "/cpsprodpb/1743E/production/_101649259_850742ef-c5c5-4537-97b1-6d0eec101534.jpg",
                   "originCode": null,
-                  "copyrightHolder": "BBC"
+                  "copyrightHolder": "GETTY IMAGES"
```

Comment:
Are these meant to be in all caps?

Reply:
@jimjohnsonrollings ^

Reply:
They will be mixed case in the data - we should however always display them in upper case.